### PR TITLE
Remove jsonld APIs and config.

### PR DESCRIPTION
**Pull request by mattcollier**
Interested in feedback wrt FIXME in util.js.

Should the code from `jsonld.hasValue` be copied here as a helper function.

It's unclear how much code that would involve since `hasValue` calls other jsonld APIs

https://github.com/digitalbazaar/jsonld.js/blob/master/lib/util.js#L212-L240

### CODE REVIEW

**Comment by dlongley** [comment]
We should just reimplement that function without using jsonld. It's just a check to see if `name` *is* a certain value or *contains* a certain value if `name`'s value is an array.

**Comment by mattcollier** [comment]
Science behind hasValue helper: https://gist.github.com/mattcollier/6f35a181e4a6f333dfdb9d78ff7b0ddc

If we can get this in `bedrock.util` we can use it in all the places.

File: lib/util.js [hunk 1]
```diff
@@ -489,3 +488,11 @@ api.callbackify = fn => {
     return fn.apply(null, args);
   };
 };
+
+// a replacement for jsonld.hasValue
+api.hasValue = (obj, key, value) => {
+  if(Array.isArray(obj[key])) {
```
Comment (dlongley) [inline lib/util.js]: Why not add `const v = obj[key];` at the top to avoid look ups on `obj[key]` three times?

Comment (mattcollier) [reply lib/util.js]: did that in the loop bench, copied over here as well now, no perceptible performance change, but code is simpler I think.

**Pull request merged**